scm/src/gmtb_scm_type_defs.f90

```diff
@@ -1509,6 +1608,22 @@ subroutine scm_state_create(scm_state, n_columns, n_levels, n_time_levels)
 
   end subroutine scm_state_create
 
+  subroutine scm_input_create_model_ics(scm_input, nlev_soil,nlev)
```

Comment:
@pjpegion I'm curious why a separate subroutine is needed for this.

Reply:
Guess It does not,  I was just trying to keep the additional allocations separate.  It could be added to the scm_input_create subroute and bracketed by an if statement.

Reply:
OK, thanks.